core/dbt/graph/selector.py

```diff
@@ -31,6 +31,8 @@ def can_select_indirectly(node):
     """
     if node.resource_type == NodeType.Test:
         return True
+    elif node.resource_type == NodeType.Unit:
+        return True
```

Comment:
My understanding is that this is to power selection for the unit test task, but does not actually include unit tests in other task execution because `UnitTestNodes` are not in the manifest. Is that correct?

Reply:
In this case the objects with NodeType.Unit are actually the UnitTestCases in manifest.unit_tests, which are in the manifest. They're not included in other tasks because each task has a Selector which filters out other resource types. So unit test cases aren't included in the "run" task job_queue because it only executes NodeType.Model.